ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/precompiles/privacy/FlexiblePrivacyPrecompiledContract.java

```diff
@@ -104,6 +104,10 @@ public PrecompileContractResult computePrecompile(
     if (skipContractExecution(messageFrame)) {
       return NO_RESULT;
     }
+    if (input == null || input.size() < 32) {
+      LOG.debug("Can not fetch private transaction payload with key {}", input);
```

Comment:
Currently we do only have two valid sizes, 32 and 64 bytes, so I guess we could do:
`if (input == null || (input.size() != 32 && input.size() != 64))`
And I think we should log that there was an invalid input and probably even log this as an error?

Reply:
done